spec/features/checkout_feature_spec.rb

```diff
@@ -14,7 +14,7 @@
   after(:all) { DatabaseCleaner.clean_with(:truncation) }
 
   context "when not logged in" do
-    it "redirects to login page if not logged in", js: true do
+    it "redirects to login page if not logged in", :redirect_login, js: true do
```

Comment:
@andela-tpeters: `redirect_login` is a tag name I gave to the particular test case to facilitate testing, rather than have the entire integration test run every time I made changes